states/networks.sls

```diff
@@ -32,10 +32,10 @@ libvirt_virsh_net_{{ name }}:
     - run
   {% if n.ensure|default('running') in ['present', 'running'] %}
     - name: virsh net-define {{ nf_path }}
-    - unless: virsh -q net-list --all | grep -q '^{{ name }}'
+    - unless: virsh -q net-list --all | grep -Eq '^\s+{{ name }}'
```

Comment:
please replace ``\s+`` by ``\s*``, to make backwards-compatiblity working (in case there's no space between ^ and {{ name }})